paddle/fluid/operators/mkldnn/transpose_mkldnn_op.cc

```diff
@@ -48,6 +48,8 @@ class TransposeMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
       return;
     }
 
+    platform::SetInMemDescWithLogicalLayoutFusesSupport(
```

Comment:
Why is it here and not before `if (ndims == 1)` line? Will it work while this `if` statement will be true?

Reply:
transpose_axis.size() need at lest 2?

Reply:
I think it should work with this case.